python/python/lancedb/rerankers/rrf.py

```diff
@@ -0,0 +1,58 @@
+import pyarrow as pa
+
+from collections import defaultdict
+from .base import Reranker
+
+
+class RRFReranker(Reranker):
+    """
+    Reranks the results using Reciprocal Rank Fusion(RRF) algorithm based
+    on the scores of vector and FTS search. For missing scores,
+    fill with `fill` value.
+    Parameters
+    ----------
+    K : int, default 60
+        A constant used in the RRF formula (default is 60). Experiments
+        indicate that k = 60 was near-optimal, but that the choice is
+        not critical. See paper:
+        https://plg.uwaterloo.ca/~gvcormac/cormacksigir09-rrf.pdf
+    return_score : str, default "relevance"
+        opntions are "relevance" or "all"
+        The type of score to return. If "relevance", will return only the relevance
+        score. If "all", will return all scores from the vector and FTS search along
+        with the relevance score.
+    """
+
+    def __init__(self, K: int = 60, return_score="relevance"):
```

Comment:
Its probably easy to confuse the constant K used in RRF with k in top-k in knn. Currently using this as the its the standard in RRF algo.